en/components/list.md

```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
+Let's do this.
+
+Let's add an input field to the top in our component template first and bind it to a property in our component called _searchContact_:
+
+```html
+<!--contacts.component.html-->
+
+<div class="igx-form-group" style="margin: 4px">
+  <input class="igx-form-group__input--search" placeholder="Search Contacts" [(ngModel)]="searchContact">
+  <label class="igx-form-group__label">
+    <igx-icon name="search"></igx-icon>
+  </label>
+</div>
+```
+
+It's time to import the **IgxFilterModule** in our **app.module.ts** file and **IgxFilterOptions** in our contacts component:
+
+```typescript
+    // app.module.ts
+    ...
+    import { IgxFilterModule } from 'igniteui-js-blocks/main';
+
+    @NgModule({
+        imports: [..., IgxFilterModule]
+    })
+
+    // contacts.component.ts
+    ...
+    import { IgxFilterOptions } from 'igniteui-js-blocks/main';
+
+    @Component({...})
+    export class ContactListComponent {
+        public searchContact: string;
+        ...
+        get filterContacts(): IgxFilterOptions {
+            const fo = new IgxFilterOptions();
+            fo.key = 'name';
+            fo.inputFiels = this.searchContact;
+            return fo;
+        }
+    }
+```
+
+After importing the `IgxFilterOptions`, we need to register a new getter method that will return the filtering options to be used by the pipe each time the `searchContact` property gets updated. For the filter to work we need to register a `key` to filter the contact object by. In our case that would be the `name` of each contact. The second property that has to be registered on the `IgxFilterOptions` object is the value that we should check against when comparing our contact name. This would be the `searchContact` property that we bound to the input field above our contacts list.
+
+Finally, we need to apply the filtering pipe to our contacts data before we can use it. So in our template we simply add:
+
+```html
+<!--contacts.component.html-->
+
+<igx-list-item *ngFor="let contact of contacts | igxFilter: filterContacts; let i = index">
+    ...
+</igx-list-item>
+```
+
+<div class="divider"></div>
+
+### API Summary
+
+In this article we covered a lot of ground with the list component. We created a list of contact items. Used some additional Ignite UI for Angular components inside our list items, like avatars and icons. Created some custom item layout and styled it. Finally, we added list filtering. The list component has a few more APIs to explore, which are listed below.
+
+####Inputs
+The following inputs are available on the **igx-list** component:
+| Name | Type | Description |
+| :--- | :--- | :--- |
+| `emptyListImage` | String | Provide an image url to be used as placeholder when the list is empty. |
+| `emptyListMessage` | String | Provide an emtpy list message. |
+| `emptyListButtonText` | String | Customize the empty list button text. |
+| `emptyListButtonText` | String | Customize the empty list button text. |
```

Comment:
Why is the same API listed twice? `emptyListButtonText`  Also, "empty" is misspelled on line 329 as "emtpy".